src/main/webapp/app/course/manage/course-update.component.ts

```diff
@@ -61,6 +62,7 @@ export class CourseUpdateComponent implements OnInit {
     faTimes = faTimes;
     faQuestionCircle = faQuestionCircle;
     faExclamationTriangle = faExclamationTriangle;
+    isAllowedToChangeOrganizations = false;
```

Comment:
I'd use a more generic name here, this way you can use the same attribute at other places in the template as well if needed.

Also please move it a bit more upwards, you paced it below the icon attributes.
```suggestion
    isAdmin = false;
```

Reply:
Thank you for your comment. I addressed it in my latest commit